src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -1179,23 +1179,26 @@ public ProtoCore.DSASM.ProcedureNode GetProcedureFromInstance(int classScope, Pr
                                 procName = Constants.kGetterPrefix + procName;
                                 procCallNode = classNode.GetFirstStaticFunctionBy(procName);
                                 isStaticCall = procCallNode != null;
-                            }
-                            else if (subPass == ProtoCore.CompilerDefinitions.SubCompilePass.None)
-                            {
-                                string message = String.Format(ProtoCore.Properties.Resources.kStaticMethodNotFound,
-                                                               className,
-                                                               procName);
+                                if (procCallNode == null)
```

Comment:
Added a check for `procCallNode` after computing it in line 1180. This check was missing earlier. In the case where it is null, we need to log a warning saying the method is not found.